libs/error-analysis/src/lib/ErrorAnalysisDashboard/Controls/FeatureList/FeatureList.tsx

```diff
@@ -123,8 +130,14 @@ export class FeatureList extends React.Component<
     this.updateSelection();
   }
 
-  public componentDidUpdate(prevProps: IFeatureListProps): void {
-    if (this.props.importances !== prevProps.importances) {
+  public componentDidUpdate(
+    prevProps: IFeatureListProps,
+    prevState: IFeatureListState
```

Comment:
we should not monitor state in component did update, this is circular refreence.
Can we do action directly when we set state?

Reply:
done!